README.markdown

```diff
@@ -22,13 +22,14 @@ Web posts are send using the HTTParty gem. The message is sent as the request bo
 
     messenger http://example.com "Message"
 
+Slack
+---
 
-Campfire
```

Comment:
I didn't, that was a copypasta mistake. I copied the Campfire example and then did an awful job of editing it to be Slackified.